components/mdx/Storybook.tsx

```diff
@@ -0,0 +1,27 @@
+import LazyLoad from 'react-lazyload'
+import { StorybookIcon } from 'components/Icons'
+
+const Storybook = ({ id, lib }) => (
+  <LazyLoad height={400} once>
+    <h3 className="text-xl mb-4 tracking-tight mt-6 heading">Example</h3>
+    <div className="mb-2">
+      <a
+        className="flex mt-3 items-center opacity-60 text-xs text-gray-700"
+        target="_blank"
+        rel="nofollow noopener noreferrer"
+        href={`https://${lib}.vercel.app/?path=/story/${id}`}
+      >
+        <StorybookIcon className="mr-1" /> Open in Storybook
```

Comment:
Let's right align interactions to the top of the story preview or the title to keep the experience consistent. I also might suggest coloring the Storybook logo if put beside the title to better communicate the hand-off.

![image](https://user-images.githubusercontent.com/23324155/115922790-f29b6880-a442-11eb-8319-9f893b658a63.png)

Reply:
Added